lib/config/migrations/custom/package-rules-migration.ts

```diff
@@ -23,6 +24,49 @@ function renameKeys(packageRule: PackageRule): PackageRule {
   return newPackageRule;
 }
 
+const dockerLanguageManagers = [
+  'ansible',
+  'dockerfile',
+  'docker-compose',
+  'droneci',
+  'kubernetes',
+  'woodpecker',
+];
+
+function removeMatchLanguage(packageRule: PackageRule): PackageRule[] {
+  const newPackageRules: PackageRule[] = [];
+  const matchLanguages = packageRule['matchLanguages'];
```

Comment:
```suggestion
  const matchLanguages = packageRule.matchLanguages;
```
use normal dot syntax